### PR TITLE
Update Dependencies and Plugins

### PR DESCRIPTION
**Pull request by stevegk**
### Before submitting a pull-request to GitBucket I have first:

- [x] read the [contribution guidelines](https://github.com/gitbucket/gitbucket/blob/master/.github/CONTRIBUTING.md)
- [x] rebased my branch over master
- [x] verified that project is compiling
- [x] verified that tests are passing
- [x] squashed my commits as appropriate

This updates all dependencies and plugins **except** blocking-slick32, Jetty and Scalatra.

It includes a fix for an api breaking change in Cache2K Final

### CODE REVIEW

File: src/main/scala/gitbucket/core/util/JGitUtil.scala [hunk 1]
```diff
@@ -189,11 +189,10 @@ object JGitUtil {
...
-      override def accept(entry: CacheEntry[String, Int]): Unit = {
-        if(entry.getKey.startsWith(keyPrefix)){
-          cache.remove(entry.getKey)
-        }
+    cache.keys().forEach(new Consumer[String] {
+      override def accept(key: String) = {
+        if (key.startsWith(keyPrefix))
+          cache.remove(key)
       }
     })
```
Comment (takezoe) [inline src/main/scala/gitbucket/core/util/JGitUtil.scala]: Thanks! But this callback can be written more simply as:
```scala
    cache.keys.forEach(key => {
      if(key.startsWith(keyPrefix)){
        cache.remove(key)
      }
    })
```

**Comment by takezoe** [comment]
@stevegk It looks good. I added a tiny comment, so I'm glad if you will fix it. 😉

**Comment by stevegk** [comment]
Fixed and rebased.

**Comment by takezoe** [comment]
@ndarilek Thanks!

**Comment by takezoe** [comment]
@stevegk Thanks!

File: project/project/plugins.sbt [hunk 2]
```diff
@@ -1 +0,0 @@
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC11")
```
Comment (xuwei-k) [inline project/project/plugins.sbt]: Why removed? This is not redundant setting.

Comment (takezoe) [reply project/project/plugins.sbt]: Exactly. I restored it in d17aed23576f9d9cec49e476b57956f4c045c580.

**Pull request merged**